webapp/src/widgets/switch.tsx

```diff
@@ -12,7 +12,7 @@ type Props = {
 
 // Switch is an on-off style switch / checkbox
 function Switch(props: Props): JSX.Element {
-    const className = props.isOn ? 'Switch on' : 'Switch'
+    const className = `Switch OverrideMain${props.isOn ? ' on' : ''}`
```

Comment:
The `OverrideMain` class souldn't be CamelCase, the `Switch` is CamelCase because is the class with the Component name itself, but you can add another class there if you want, but it needs to be lowercase, and problably with dashes.

Do you mind to rename this to `override-main`?

Reply:
Renamed. Where can I read about these naming rules?

Reply:
I don't think is written anywhere, probably it should be documented anywhere.